examples/metrics-basic/src/main.rs

```diff
@@ -6,7 +6,12 @@ use std::error::Error;
 use std::vec;
 
 fn init_meter_provider() -> opentelemetry_sdk::metrics::SdkMeterProvider {
-    let exporter = opentelemetry_stdout::MetricsExporterBuilder::default().build();
+    let exporter = opentelemetry_stdout::MetricsExporterBuilder::default()
+        // Build exporter using Default (Cumulative) Temporality Selector.
+        .with_default_temporality_selector()
```

Comment:
I think a simpler interface would be to just have one method `with_temporality` which takes in an enum value. No need to have separate methods for delta, default, cumulative. Something like this:
```rust
.with_temporality(Temporality::Delta)
// or 
.with_temporality(Temporality::Cumulative)
```

If the user doesn't explicitly provide the temporality information using `with_temporality` we just use the default.

Reply:
+1

Reply:
I was thinking along those lines also, this would prevent the use of the `LowMemory` option, if that's a fair trade off I can go ahead with this

Reply:
we don't need to prevent LowMemory. It can be added to the Enum. Either right away or a future addition

Reply:
I switched over to using the `Temporality` enum directly